apps/verticalhome/index.html

```diff
@@ -68,50 +73,56 @@
     <link rel="stylesheet" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <!-- <script defer src="shared/elements/gaia-header/dist/gaia-header.js"></script> -->
 
+    <script defer src="js/pin_app_navigation.js"></script>
+    <script defer src="js/more_apps_navigation.js"></script>
+    <script defer src="js/clock.js"></script>
     <script defer src="js/app.js"></script>
-    <script defer src="js/search.js"></script>
+<!--    <script defer src="js/search.js"></script> -->
     <!-- <script defer src="shared/js/icc_helper.js"></script> -->
     <!-- <script defer src="shared/js/version_helper.js"></script> -->
     <!-- <script defer src="js/configurator.js"></script> -->
   </head>
   <body class="theme-media">
     <div id="curtain"></div>
+    <section id="main-screen" role="region">
 
-    <section id="edit-header" role="region">
-      <gaia-header>
-        <h1 data-l10n-id="edit-mode"></h1>
-        <a href="#" data-l10n-id="edit-done" id="exit-edit-mode"></a>
-      </gaia-header>
+      <div id="clock">
+        <div id="clock-time"></div>
+        <div id="date"></div>
+      </div>
+
+      <div id="pin-apps-list">
+
+        <div id ='moreApps' data-index='999' class='pin-app-item'>
+          <input id="moreAppsButton" type="button"/>
+          <span class="title">More Apps</span>
+        </div>
+      </div>
     </section>
 
-    <div id="search" role="button" aria-labelledby="search-input" aria-haspopup="true">
-      <form id="search-form">
-        <input id="search-input" type="text" x-inputmode="verbatim" role="presentation"
-          placeholder="Search or enter address" data-l10n-id="search-or-enter-address" value="" dir="auto">
-        </input>
-      </form>
-    </div>
-
-    <gaia-grid id="icons" dragdrop></gaia-grid>
-
-    <gaia-menu id="contextmenu-dialog" hidden>
-      <button id="homescreen-settings-action" data-l10n-id="homescreen-settings" type="button"></button>
-      <button id="change-wallpaper-action" data-l10n-id="change-wallpaper" type="button"></button>
-      <button id="create-smart-collection" data-l10n-id="add-smart-collections" type="button"></button>
-    </gaia-menu>
-
-    <section id="edit-group" role="region" hidden>
-      <gaia-header id="edit-group-header" action="close">
-        <h1 data-l10n-id="edit-group-header"></h1>
-        <button id="edit-group-save" disabled type="button" data-l10n-id="edit-done"></button>
-      </gaia-header>
-
-      <form>
-        <p>
-          <input type="text" id="edit-group-title" x-inputmode="latin-prose" dir="auto"></input>
-          <button id="edit-group-title-clear" type="reset"></button>
-        </p>
-      </form>
+    <section id="more-apps-screen" role="region" hidden="">
+      <div class='more-apps-header'>More Apps</div>
+      <gaia-grid-rs id="icons"></gaia-grid-rs>
```

Comment:
Since you're only using the gaia-grid for the more apps, you can dramatically improve the homescreen startup performance by not loading all of the gaia-grid code until it is actually needed.